docs/topics/api/hero.rst

```diff
@@ -15,13 +15,15 @@ Combined Hero Shelves
 
 .. _hero-shelves:
 
-This convienence endpoint serves a single, randomly selected, primary hero shelf.
+This convienence endpoint serves a single, randomly selected, primary hero shelf,
+and a single, randomly selected secondary hero shelf.
 
 
 .. http:get:: /api/v4/hero/
 
     :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
     :>json object primary: A :ref:`primary hero shelf <primary-hero-shelf>`.
+    :>json object secondary: A :ref:`primary hero shelf <primary-hero-shelf>`.
```

Comment:
I think you wanted to link to the **secondary** hero shelf item here

Reply:
https://github.com/mozilla/addons-server/pull/11988/commits/7ea155b54d52a51a8e9f29235d787574c59cac66